api/base/serializers.py

```diff
@@ -353,10 +353,12 @@ def process_related_counts_parameters(self, params):
         if utils.is_truthy(params) or utils.is_falsy(params):
             return params
 
+        is_retraction = getattr(value, 'is_retracted', False)
```

Comment:
This doesn't seem like the ideal approach--`Relationship` is meant to be a generic field (not coupled to OSF), and we're adding registration retraction logic here. There may be other times when we'll need to exclude counts (deleted projects, etc.).

Can you come up with a more general solution?